src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage handles the reading and writing of the TaskList to an output file.
+ *
+ * @author Alvin Jiang Min Jun
+ * @version v0.1
+ */
+public class Storage {
+
+    String filePath;
+    final static String PATH = "./data";
+
+    /**
+     * Storage constructor that creates an instance of a Storage object.
+     *
+     * @param filePath The file path to which the storage can read and write files.
+     */
+    public Storage(String filePath) {
```

Comment:
Is this file path OS-independent as required? Could try using Paths.get() to fetch the relative file path then appending the input filePath to form the full file path for storage.